AppAuth-JS/built/node_support/node_request_handler.d.ts

```diff
@@ -0,0 +1,12 @@
+import { RandomGenerator } from '../crypto_utils';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork